packages/components/organisms/f-registration/src/components/Registration.vue

```diff
@@ -137,23 +137,29 @@
             </form>
             <p :class="$style['c-registration-link']">
                 {{ copy.navLinks.termsAndConditions.prefix }}
-                <a
-                    class="o-link--bold"
+                <v-link
+                    is-bold
                     data-test-id="ts-and-cs-link"
                     :href="copy.navLinks.termsAndConditions.url"
-                    target="_blank">{{ copy.navLinks.termsAndConditions.text }}</a>{{ copy.navLinks.termsAndConditions.suffix }}
+                    target="_blank">
```

Comment:
Should this be `:opensInNewLocation` instead of straight target for aria-description to be added? Here and below

Reply:
This prop has been removed in 0.4.0. The `target="_blank"` that adds the aria description automatically